src/components/editor/property/InputURI.jsx

```diff
@@ -79,7 +79,7 @@ const InputURI = (props) => {
     return errors
   }
 
-
+  console.log('items', props.items)
```

Comment:
stray debug statement?  (will test out locally per instructions and leave an actual review in a bit)